From c486b6fc7f3f23240a278ef4da91d208c06a4735 Mon Sep 17 00:00:00 2001
From: Pratham Deshmukh <p-deshmukh@ti.com>
Date: Thu, 12 Feb 2026 17:23:40 +0530
Subject: [PATCH 2/5] Try to find package Protobuf before checking if it was
 found

Upstream-Status: Pending

Signed-off-by: Pratham Deshmukh <p-deshmukh@ti.com>
---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8593fe4a3..9fed84add 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -142,6 +142,8 @@ if(ONNX_BUILD_TESTS)
   set(googletest_STATIC_LIBRARIES GTest::gtest)
 endif()
 
+find_package(Protobuf)
+
 if(NOT ONNX_BUILD_CUSTOM_PROTOBUF)
 if((ONNX_USE_LITE_PROTO AND TARGET protobuf::libprotobuf-lite) OR ((NOT ONNX_USE_LITE_PROTO) AND TARGET protobuf::libprotobuf))
   # Sometimes we need to use protoc compiled for host architecture while linking
-- 
2.34.1

